build/cqs.tsv

```diff
@@ -270,6 +272,8 @@ org.springframework.security:spring-security-core	4.2.7.RELEASE	provided
 org.springframework.security:spring-security-web	4.2.7.RELEASE	provided
 org.springframework:spring-web	4.3.18.RELEASE	provided
 
+io.confluent:kafka-schema-registry	5.1.0	test
+io.confluent:kafka-schema-registry	tests:5.1.0	test
```

Comment:
@jnh5y we don't need to do anything for these test scope dependencies, right?

Reply:
Yeah, test scope we should be able to skip on doing anything.